### DISCUSSION TITLE
Terraform deploy error

### DISCUSSION THREAD

**Discussion by MauricioOlitel** [twilio-professional-services/flex-project-template#582] [General]
I'm having an error when executing the deploy, on my system it's giving an error in terraform

twilio_taskrouter_workspaces_task_channels_v1.voice: Refreshing state... [id=WSc43b31c20a726f3a9447f2704f0acd19/TCbf5ed74ea40e80577d13d56dd13b7247]
╷
│ Error: Invalid value for variable
│ 
│   on /home/runner/work/Corteva/Corteva/infra-as-code/terraform/environments/default/variables.tf line 83:
│   83: variable "SCHEDULE_MANAGER_ENV_SID" {
│     ├────────────────
│     │ var.SCHEDULE_MANAGER_ENV_SID is "<YOUR_SCHEDULE_MANAGER_ENV_SID>"
│ 
│ SCHEDULE_MANAGER_ENV_SID expected to start with "ZE".
│ 
│ This was checked by the validation rule at
│ /home/runner/work/Corteva/Corteva/infra-as-code/terraform/environments/default/variables.tf:86,3-13.
╵

╷
│ Error: Invalid value for variable
│ 
│   on /home/runner/work/Corteva/Corteva/infra-as-code/terraform/environments/default/variables.tf line 92:
│   92: variable "SCHEDULE_MANAGER_CHECK_FUNCTION_SID" {
│     ├────────────────
│     │ var.SCHEDULE_MANAGER_CHECK_FUNCTION_SID is "<YOUR_SCHEDULE_MANAGER_CHECK_FUNCTION_SID>"
│ 
│ SCHEDULE_MANAGER_CHECK_FUNCTION_SID expected to start with "ZH".
│ 
│ This was checked by the validation rule at
│ /home/runner/work/Corteva/Corteva/infra-as-code/terraform/environments/default/variables.tf:[95](https://github.com/MauricioOlitel/Corteva/actions/runs/9651274328/job/26618909129#step:6:96),3-13.
╵

Error: Terraform exited with code 1.
Error: Process completed with exit code 1.

Do you have any ideas?

**Comment by dremin** [comment]
Hi @MauricioOlitel! It looks like the variables file isn't getting its placeholder strings replaced, which is performed by the postinstall script in the provided scripts [here](https://github.com/twilio-professional-services/flex-project-template/blob/main/infra-as-code/state/import_internal_state.sh#L95). The script makes an environment-specific copy of the example file (if it doesnt exist) then replaces the placeholder values. By chance have you modified these files or scripts?

**Comment by jvidamins** [comment]
I have the same errors on my install. Do we have a fix to this yet?
